app/models/components/course/achievements_ability_component.rb

```diff
@@ -23,6 +24,10 @@ def allow_staff_manage_achievements
     can :manage, Course::Achievement, course_staff_hash
   end
 
+  def do_not_allow_staff_award_automatically_awarded_achievements
+    cannot :award, Course::Achievement, manually_awarded?: false
```

Comment:
This is potentially iffy because it can be confused with a database call, which will return a SQL error.